src/config/entities/configuration.ts

```diff
@@ -31,6 +31,14 @@ export default () => ({
     level: process.env.LOG_LEVEL || 'debug',
     silent: process.env.LOG_SILENT?.toLowerCase() === 'true',
   },
+  notFoundErrorTTLSeconds: {
+    contract: parseInt(
+      process.env.CONTRACT_NOT_FOUND_ERROR_TTL_SECONDS ?? `${60 * 60}`,
+    ),
+    token: parseInt(
+      process.env.TOKEN_NOT_FOUND_ERROR_TTL_SECONDS ?? `${60 * 60}`,
+    ),
+  },
```

Comment:
I'm setting 1 hour here as the default for both, but this is open to discussion.

Reply:
Agreed, I'll set these defaults to 60 seconds as well then.

Reply:
Changed in [f64c4da](https://github.com/safe-global/safe-client-gateway-nest/pull/578/commits/f64c4dad0e26f2530a411001a900ee3b255cc940)